Arrowgene.Ddon.GameServer/Handler/InstanceEnemyGroupEntryHandler.cs

```diff
@@ -18,22 +19,20 @@ public InstanceEnemyGroupEntryHandler(DdonGameServer server) : base(server)
 
         public override void Handle(GameClient client, StructurePacket<C2SInstanceEnemyGroupEntryNtc> packet)
         {
-            var layout = packet.Structure.LayoutId;
+            CDataStageLayoutId layout = packet.Structure.LayoutId;
 
             //Somehow we've entered twice.
-            if (client.Character.EnemyLayoutOwnership.ContainsKey(layout.AsTuple()))
+            if (client.Character.EnemyLayoutOwnership.ContainsKey(layout))
             {
-                Logger.Debug($"{client.Character.FirstName} double entry at {layout}.");
                 //return;
```

Comment:
Dead code?

Reply:
If nothing is being done then it's probably better to just remove the if entirely. It can be added back later if ever needed